src/CSnakes.SourceGeneration/Parser/PythonParser.Constants.cs

```diff
@@ -74,7 +74,7 @@ from close in Character.EqualTo('\'')
     public static TokenListParser<PythonToken, PythonConstant.HexidecimalInteger> HexidecimalIntegerConstantTokenizer { get; } =
         Token.EqualTo(PythonToken.HexidecimalInteger)
         .Select(d => new PythonConstant.HexidecimalInteger(long.Parse(d.ToStringValue().Substring(2).Replace("_", ""), NumberStyles.HexNumber)))
```

Comment:
This misspelling is all over the place, will fix in another PR

Reply:
So revert it in this PR?